lib/utils/rename.js

```diff
@@ -1,16 +1,182 @@
 'use strict'
 var fs = require('graceful-fs')
+var path = require('path')
+var rimraf = require('rimraf')
 var SaveStack = require('./save-stack.js')
 
 module.exports = rename
 
 function rename (from, to, cb) {
   var saved = new SaveStack(rename)
```

Comment:
Its from the original `rename` - I believe it makes sure that the stack trace includes the call to this function (although tbh I'm not entirely sure)